server/server.go

```diff
@@ -38,5 +37,21 @@ func main() {
 		return c.JSON(http.StatusOK, res)
 	})
 
+  
+  type Content struct {
+    Username string `json:"username"`
```

Comment:
We only need the content body in this struct.  A username is not part of the content and the content id won't exist until after the POST request has been executed, as a POST is going to create a new resource.